README.md

```diff
@@ -102,11 +102,6 @@ colors:
   bat_theme: "ansi"
 ```
 
-Any configuration values can be overridden by passing them as arguments to the CLI.
-When passing arguments to the CLI, replace `_` with `-`, e.g. `preview_with_bat`
-should be passed to the CLI as `--preview-with-bat`.
-See `ctrlg --help` and `ctrlg find --help`.
```

Comment:
I decided to remove this behavior because the implementation was unmaintainable. I will add it back once I figure out a better way.